editor/src/clj/editor/model_scene.clj

```diff
@@ -261,14 +251,31 @@
 (defn- doubles->floats [ds]
   (float-array (map float ds)))
 
+(defn- bytes-to-indices [^ByteString indices index-format]
+  (let [num-bytes (.size indices)
+        ba (byte-array num-bytes)
+        _ (.copyTo indices ba 0)
+        bb (ByteBuffer/wrap ba)
```

Comment:
DIdn't manage to get a readable byte byffer from the com.google.protobuf.ByteString, so this copy had to be done for now.